x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/state/overview/effects.ts

```diff
@@ -21,3 +28,49 @@ export function* fetchMonitorOverviewEffect() {
     )
   );
 }
+
+export const TRENDS_CHUNK_SIZE = 50;
```

Comment:
It is good to keep this set to around 40+ ideally, because on most screens where there are enough for the chunk size to matter, the first 40-or-so monitors are getting rendered. After we switched to using `msearch` this seems to be no issue from a perf perspective.